gnosis/eth/oracles/oracles.py

```diff
@@ -544,6 +546,48 @@ def get_price(self, token_address: str) -> float:
             )
 
 
+class SuperfluidOracle(PriceOracle):
```

Comment:
Please create the oracle in `gnosis/eth/oracles/superfluid.py` , we are doing that for the new oracles and will be migrating the old ones, that way it's easier to maintain